transport-classes-epoll/src/main/java/io/netty/channel/epoll/NativeArrays.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2025 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.channel.epoll;
+
+import io.netty.channel.unix.IovArray;
+
+final class NativeArrays {
```

Comment:
I noticed there's nothing epoll specific in `NativeDatagramPacketArray`, so potentially it and this class could live in `unix-common` and be shared with kqueue. Not a hill I'll die on, though, if you disagree.

Reply:
(though I suppose `NativeDatagramPacketArray` is Linux/glibc specific)

Reply:
Yep that's why its here...